sentry-sample/src/main/AndroidManifest.xml

```diff
@@ -2,7 +2,7 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:tools="http://schemas.android.com/tools"
     package="io.sentry.sample">
-
+  <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
```

Comment:
Should this be set in `android-core` instead?
Since we'll be checking the network status before sending event?

Reply:
we can also do that once we implement the check btw*

Reply:
I'd make the user to opt-in for that, if the user adds the permission, all good.
the check is already done https://github.com/getsentry/sentry-android/blob/master/sentry-android-core/src/main/java/io/sentry/android/core/util/Permissions.java